python/ambassador/config/resourcefetcher.py

```diff
@@ -151,8 +164,7 @@ def load_from_filesystem(self, config_dir_path, recurse: bool=False,
             self.finalize()
 
     def parse_yaml(self, serialization: str, k8s=False, rkey: Optional[str]=None,
-                   filename: Optional[str]=None, finalize: bool=True, namespace: Optional[str]=None,
-                   metadata_labels: Optional[Dict[str, str]]=None) -> None:
+                   filename: Optional[str]=None, finalize: bool=True, metadata_labels: Optional[Dict[str, str]]=None) -> None:
```

Comment:
I removed the namespace arg from this method and its callees because it wasn't called with it anywhere that I could find and propagating it to the new resource processing code felt convoluted.